lib/components/footer.dart

```diff
@@ -0,0 +1,145 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_web/responsive.dart';
+
+import '../constants.dart';
+
+class Footer extends StatelessWidget {
+  const Footer({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return (!isMobile(context)) ? DesktopFooter() : MobileFooter();
+  }
+}
+
+class DesktopFooter extends StatelessWidget {
+  const DesktopFooter({
+    Key key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: EdgeInsets.symmetric(vertical: 20, horizontal: 40),
+      child: Row(
+        children: <Widget>[
+          Expanded(
+            flex: 1,
+            child: Text(
+              'All Right Reserved',
+              style: TextStyle(
+                fontSize: 10
+              ),
+            )
+          ),
+
+          
+            Expanded(
```

Comment:
Formatting isssue here (this `Expanded` is one tab more at right than previous one)